react-frontend/src/components/DigitalFramework/digitalframeworktable.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import { Col, Row, Grid } from 'react-flexbox-grid';
+
+const DigitalFrameworkTable = (props) => {
+  const initial = window.innerWidth < 800;
+  const [isMobile, setIsMobile] = useState(initial);
+
+  useEffect(() => {
+    function handleResize() {
```

Comment:
can this be replaced with a `media query` in the css? Have any questions? let us know :)

Reply:
There needs to be value changes in state to modify the props of the antd tables

Reply:
hmmm. This still seems funny. There should be a better way.